modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -911,6 +911,108 @@ definition {
 		}
 	}
 
+	@description = "LPS-154872 - Verify that it's possible to filter and sort entries using metadata fields"
+	@priority = "4"
+	test CanFilterAndSortEntriesViaMetadataFields {
+		property portal.upstream = "quarantine";
```

Comment:
If we're adding a test to quarantine, we should create a ticket to add the test to quarantine and commit under that ticket. The ticket should have an explanation why it was quarantined so we don't forget in the future. It really helps when we start auditing our quarantined tests.

Reply:
I created a new ticket for fix: [LPS-164870](https://issues.liferay.com/browse/LPS-164870)